esp_pixel_driver.ino

```diff
@@ -10,16 +10,29 @@
 #include <WiFi.h>
 #endif
 
+#include <ArduinoJson.h>
+#include <EEPROM.h>
+
+#include "config.h"
+#include "memory.h"
+#include "wifimanager_adapter.h"
+#include "pixel_server.h"
+
 //LEDs
-CRGB leds[STRIP_LENGTH];
+CRGB *leds;
```

Comment:
Moving to pointers for this and the universesReceived array so we can take strip length in from config